Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gcc:4.9.4
+# assumes this alread has git installed
+#RUN apt-get -y update && apt-get install -y git
+RUN mkdir /opt/externpro
+COPY externpro-16.01.1-gcc494-64-Linux.sh /opt/externpro/
```

Comment:
where does externpro have to live?  You could clone it and build it the same as is done with ovsrpro if you wanted...